src/ports/postgres/modules/deep_learning/madlib_keras.sql_in

```diff
@@ -1750,6 +1750,20 @@ CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.madlib_keras_fit(
 $$ LANGUAGE sql VOLATILE
 m4_ifdef(`__HAS_FUNCTION_PROPERTIES__', `MODIFIES SQL DATA');
 
+CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.madlib_keras_generate_mst_table(
+    model_selection_table   VARCHAR,
+    model_arch_table        VARCHAR,
+    model_arch_id_list      INTEGER[],
+    compile_params_list     VARCHAR[],
+    fit_params_list         VARCHAR[]
+) RETURNS VOID AS $$
+    PythonFunctionBodyOnly(`deep_learning', `madlib_keras_model_hopper_utilities')
+    with AOControl(False):
+        mst_generator = madlib_keras_model_hopper_utilities.GenerateMstTable(**globals())
+        mst_generator.generate()
+$$ LANGUAGE plpythonu VOLATILE
+m4_ifdef(`__HAS_FUNCTION_PROPERTIES__', `MODIFIES SQL DATA', `');
+
```

Comment:
Can we add some user docs for this function, or is it planned for later?

Reply:
Yes it's for later